qa/rpc-tests/preciousblock.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python2
```

Comment:
Needs to be changed to python3 after rebase.

Also some other adjustments are needed in this file.